packages/by-macros/src/sql_model.rs

```diff
@@ -442,6 +455,25 @@ pub fn parse_field_attr(field: &Field) -> SqlAttributes {
                                         operations.push(auto);
                                     }
                                 }
+                                OpenedOffset::TargetTable => {
+                                    let target = match id.as_str() {
+                                        "join" => TargetTable::Join,
+                                        "foreign" => TargetTable::Foreign,
+                                        _ => {
+                                            tracing::error!("invalid target table: {id}");
```

Comment:
```suggestion
                                            panic!("target_table mut be either join or foreign: {id}");
```
panic can prevent  mis-used